duckduckgo/Android
# Optimise app onboarding dialog

Task/Issue URL: https://app.asana.com/0/0/1203887621832726/f

### Description
Create new dax dialog with a trackers blocked animations which will be used for an experiment to optimise app onboarding

### Steps to test this PR

Asana task with instructions: https://app.asana.com/0/0/1204008611485984/f


_Pixels_
- When new dialog appears -> ☑️ Check pixel `m_odc_s` is fired
- When tapping on dax dialog button -> ☑️ Check pixel `m_odc_ok` is fired
- When tapping on privacy shield modal -> ☑️ Check new pixel `m_onboarding_privacy_shield_button` is fired

### UI changes
| 🌙 Dark Mode  | 🌞 Light Mode |
| ------ | ----- |
https://user-images.githubusercontent.com/20798495/221184911-c2788461-4a15-41df-bf21-8006630d54ef.mp4|https://user-images.githubusercontent.com/20798495/221184929-b198a35a-bd78-4fb0-bb7b-7b2583c5d1d0.mp4
